mep/people/management/commands/export_creators.py

```diff
@@ -0,0 +1,24 @@
+"""
+Manage command to export creator data for use by others.
+
+Generates a CSV and JSON file including details on every creator
+database, with summary details and coordinates for associated addresses.
+
+"""
+
+from mep.people.models import Person
+from mep.books.models import Creator
+from mep.people.management.commands.export_members import Command as ExportMemberCommand
+
+
+class Command(ExportMemberCommand):
+    """Export member data."""
+
+    def get_queryset(self):
+        """filter to creators"""
+        creator_ids = {c.person_id for c in Creator.objects.all()}
+        return Person.objects.filter(pk__in=creator_ids)
```

Comment:
should be able to get this in a single query; try this:
```suggestion
        return Person.objects.filter(creator__isnull=False).distinct().count()
```

The distinct is needed because without it we get one person record for every time they have a creator entry